### DISCUSSION TITLE
Onboarding pipeline problem - created multiple accounts with same handle

### DISCUSSION THREAD

**Discussion by WhySere** [mastodon/mastodon#25734] [Q&A]
Hello!
As you may imagine, I'm brand new on mastodon.

I got confused and carried away, and **created multiple accounts on different instances**, with the same handle.
I am aware that I will not be able to re-use the same handles on instances, were I to delete the accounts there.
_Is there any way to migrate my (barely existent) data from the other accounts, to unify them?_

In any case, I did an oopsie and am aware of it, but if I did this by mistake, misunderstanding the functionality of instances, I can clearly imagine other people doing the same. **This could result in a lot of work down the line, if not addressed prior.**

Regards,
WhySere

**Comment by renchap** [comment]
You can use the account migration feature to indicate that an account moved to another server: https://blog.joinmastodon.org/2019/06/how-to-migrate-from-one-server-to-another/

**Reply by WhySere** [reply]
Thank you! I am aware of the migration feature, but I'm wondering if that will unify the accounts, or create problems because there are multiple accounts with the same handle

**Reply by renchap** [reply]
It will show the redirect banner on the accounts you migrate, which will all point to your new account.

Each server is independent, so there wont be any issues with your using the same username on multiple servers.

You can also simply delete your accounts on the servers you dont want to keep, if they dont have any followers you want to migrate.

**Reply by WhySere** [reply]
Okay that's interesting. So your suggestion would be to migrate them to the main account and then delete them?
What if later I would want to migrate my main account to the servers where my previous accounts (and thus handles) existed, would I still be able to do that?

**Reply by renchap** [reply]
Yes you should be able to do that. You can delete the accounts or simply leave them as migrated, if you want to be able to re-use them someday.

**Reply by WhySere** [reply]
That's a relief! Thank you for the support. I'll keep posting here to give updates when I'll try the solutions

**Reply by WhySere** [reply]
Alright! I confirm that migration from both profiles has been possible, and that they are currently inactive but still exist.
Thank you Renchap 👍 :)
